app/helpers/foreman_host_reports/reports_dashboard_helper.rb

```diff
@@ -0,0 +1,70 @@
+module ForemanHostReports
+  module ReportsDashboardHelper
+    def host_reports_searchable_links(name, search, counter, format)
+      content_tag :li do
+        content_tag(:span, raw('&nbsp;'), :class => 'label', :style => "background-color:" + host_reports_report_color[counter]) +
+          raw('&nbsp;') +
+          link_to(name, hosts_path(:search => search), :class => "dashboard-links") +
+          content_tag(:h4, send(counter, format))
+      end
+    end
+
+    def host_reports_report_color
+      {
+        :change_hosts => "#4572A7",
+        :failure_hosts => "#AA4643",
+        :nochange_hosts => "#DB843D",
+        :disabled_hosts => "#92A8CD",
+      }
+    end
+
+    def host_reports_get_overview(options = {})
+      format = options[:format]
+      state_labels = {
+        change_hosts: _('Change'),
+        failure_hosts: _('Failure'),
+        nochange_hosts: _('No Change'),
+        disabled_hosts: _('Disabled alerts'),
+      }
+
+      {
+        data: state_labels.map { |key, label| [label, send(key, format), host_reports_report_color[key]] },
+        searchUrl: hosts_path(search: '~VAL~'),
+        searchFilters: state_labels.each_with_object({}) do |(key, filter), filters|
+          filters[filter] = send(key, format)
+        end,
+      }
+    end
+
+    def latest_reports
+      hosts = Host.authorized(:view_hosts, Host)
+      @latest_reports = HostReport.authorized(:view_host_reports).my_reports
```

Comment:
Setting an instance value in a helper is not great, because you might override value from controller and it is hard to track down where the value gets overriden (especially when the code that gets called is decided by what widgets you have on the dashboard).
Do we need to set the variable here?

Reply:
In that case, it is redundant. 
Thank you